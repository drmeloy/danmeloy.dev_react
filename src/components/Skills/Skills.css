.Skills {
  padding-left: 130px;
  min-height: 100vh;

  p {
    font-family: 'Raleway', sans-serif;
    font-size: 2rem;
    width: 85vw;
    padding: 2% 5%;
    line-height: 1.3;
    color: white;
  }

  @media (min-width: 10px) and (max-width: 720px){
    p {
      font-size: 1em;
    }
  }

  @media (min-width: 721px) and (max-width: 1025px) {
    p {
      font-size: 1.5em;
    }
  }

  @media (max-width: 1025px){
    padding-left: 0;
  }

  background: linear-gradient(230deg, rgba(93, 93, 93, 0.03) 0%, rgba(93, 93, 93, 0.03) 50%,rgba(78, 78, 78, 0.03) 50%, rgba(78, 78, 78, 0.03) 100%),linear-gradient(107deg, rgba(55, 55, 55, 0.01) 0%, rgba(55, 55, 55, 0.01) 50%,rgba(170, 170, 170, 0.01) 50%, rgba(170, 170, 170, 0.01) 100%),linear-gradient(278deg, rgba(16, 16, 16, 0.03) 0%, rgba(16, 16, 16, 0.03) 50%,rgba(24, 24, 24, 0.03) 50%, rgba(24, 24, 24, 0.03) 100%),linear-gradient(205deg, rgba(116, 116, 116, 0.03) 0%, rgba(116, 116, 116, 0.03) 50%,rgba(0, 0, 0, 0.03) 50%, rgba(0, 0, 0, 0.03) 100%),linear-gradient(150deg, rgba(5, 5, 5, 0.03) 0%, rgba(5, 5, 5, 0.03) 50%,rgba(80, 80, 80, 0.03) 50%, rgba(80, 80, 80, 0.03) 100%),linear-gradient(198deg, rgba(231, 231, 231, 0.03) 0%, rgba(231, 231, 231, 0.03) 50%,rgba(26, 26, 26, 0.03) 50%, rgba(26, 26, 26, 0.03) 100%),linear-gradient(278deg, rgba(89, 89, 89, 0.03) 0%, rgba(89, 89, 89, 0.03) 50%,rgba(26, 26, 26, 0.03) 50%, rgba(26, 26, 26, 0.03) 100%),linear-gradient(217deg, rgba(28, 28, 28, 0.03) 0%, rgba(28, 28, 28, 0.03) 50%,rgba(202, 202, 202, 0.03) 50%, rgba(202, 202, 202, 0.03) 100%),linear-gradient(129deg, rgba(23, 23, 23, 0.03) 0%, rgba(23, 23, 23, 0.03) 50%,rgba(244, 244, 244, 0.03) 50%, rgba(244, 244, 244, 0.03) 100%),linear-gradient(90deg, rgb(1, 64, 149),rgb(198, 5, 91))
}

.Icons {
  display: grid;
  place-items: center;
  grid-template: auto / repeat(4, 1fr);
  row-gap: 3rem;

  p {
    border-radius: 50%;
    height: 15vh;
    width: 17vh;
    background: #ffffff17;
    text-align: center;
    font-family: 'Raleway', sans-serif;
    color: white;
    font-size: 1.2rem;
    padding: 1%;
    padding-top: 0;
  }

  img {
    height: 8vh;
    margin-top: 1em;
  }

  @media (min-width: 10px) and (max-width: 720px){
    row-gap: 1em;
    grid-template: auto / repeat(2, 1fr);

    p {
      font-size: .6em;
    }
  }

  @media (min-width: 721px) and (max-width: 1025px) {
    grid-template: auto / repeat(3, 1fr);
  }
}