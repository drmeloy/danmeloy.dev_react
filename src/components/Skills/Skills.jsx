import React from 'react';
import styles from './Skills.css';

export default function Skills(){
  return (
    <>
      <section className={styles.Skills}>
        <p>Technologies I currently use:</p>
        <div className={styles.Icons}>
          <p>
            <img src='../../../public/assets/js.png' alt='javascript icon'/>
            <br/>Javascript
          </p>
          <p>
            <img src='../../../public/assets/html.png' alt='html icon'/>
            <br/>HTML
          </p>
          <p>
            <img src='../../../public/assets/css.png' alt='css icon'/>
            <br/>CSS
          </p>
          <p>
            <img src='../../../public/assets/node.png' alt='node icon'/>
            <br/>Node.js
          </p>
          <p>
            <img src='../../../public/assets/react.png' alt='react icon'/>
            <br/>React
          </p>
          <p>
            <img src='../../../public/assets/redux.png' alt='redux icon'/>
            <br/>Redux
          </p>
          <p>
            <img src='../../../public/assets/mongodb.png' alt='mongodb icon'/>
            <br/>MongoDB
          </p>
          <p>
            <img src='../../../public/assets/postgres.png' alt='postgres icon'/>
            <br/>PostgreSQL
          </p>
          <p>
            <img src='../../../public/assets/github.png' alt='github icon'/>
            <br/>Github
          </p>
          <p>
            <img src='../../../public/assets/heroku.png' alt='heroku icon'/>
            <br/>Heroku
          </p>
          <p>
            <img src='../../../public/assets/jest.png' alt='jest icon'/>
            <br/>Jest
          </p>
          <p>
            <img src='../../../public/assets/shell.png' alt='shell icon'/>
            <br/>Shell
          </p>
        </div>
        <br/>
        <p>Technologies I am currently learning:</p>
        <div className={styles.Icons}>
          <p>
            <img src='../../../public/assets/graphql.png' alt='graphql icon'/>
            <br/>GraphQL
          </p>
          <p>
            <img src='../../../public/assets/typescript.png' alt='typescript icon'/>
            <br/>Typescript
          </p>
        </div>
      </section>
    </>
  );
}
