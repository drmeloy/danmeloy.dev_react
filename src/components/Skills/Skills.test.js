import React from 'react';
import { shallow } from 'enzyme';
import Skills from './Skills';

describe('Skills component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Skills />);
    expect(wrapper).toMatchSnapshot();
  });
});
