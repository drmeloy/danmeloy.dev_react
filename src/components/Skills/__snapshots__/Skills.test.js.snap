// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills component matches snapshot 1`] = `
<Fragment>
  <section
    className="Skills"
  >
    <p>
      Technologies I currently use:
    </p>
    <div
      className="Icons"
    >
      <p>
        <img
          alt="javascript icon"
          src="/public/assets/js.png"
        />
        <br />
        Javascript
      </p>
      <p>
        <img
          alt="html icon"
          src="/public/assets/html.png"
        />
        <br />
        HTML
      </p>
      <p>
        <img
          alt="css icon"
          src="/public/assets/css.png"
        />
        <br />
        CSS
      </p>
      <p>
        <img
          alt="node icon"
          src="/public/assets/node.png"
        />
        <br />
        Node.js
      </p>
      <p>
        <img
          alt="react icon"
          src="/public/assets/react.png"
        />
        <br />
        React
      </p>
      <p>
        <img
          alt="redux icon"
          src="/public/assets/redux.png"
        />
        <br />
        Redux
      </p>
      <p>
        <img
          alt="mongodb icon"
          src="/public/assets/mongodb.png"
        />
        <br />
        MongoDB
      </p>
      <p>
        <img
          alt="postgres icon"
          src="/public/assets/postgres.png"
        />
        <br />
        PostgreSQL
      </p>
      <p>
        <img
          alt="github icon"
          src="/public/assets/github.png"
        />
        <br />
        Github
      </p>
      <p>
        <img
          alt="heroku icon"
          src="/public/assets/heroku.png"
        />
        <br />
        Heroku
      </p>
      <p>
        <img
          alt="jest icon"
          src="/public/assets/jest.png"
        />
        <br />
        Jest
      </p>
      <p>
        <img
          alt="shell icon"
          src="/public/assets/shell.png"
        />
        <br />
        Shell
      </p>
    </div>
    <br />
    <p>
      Technologies I am currently learning:
    </p>
    <div
      className="Icons"
    >
      <p>
        <img
          alt="graphql icon"
          src="/public/assets/graphql.png"
        />
        <br />
        GraphQL
      </p>
      <p>
        <img
          alt="typescript icon"
          src="/public/assets/typescript.png"
        />
        <br />
        Typescript
      </p>
    </div>
  </section>
</Fragment>
`;
